### PR TITLE
[project-base] fixed tablet account popups

### PR DESCRIPTION
**Pull request by chlebektomas**
#### Description, the reason for the PR
Changed logic for showing popup and drawer for account/login.

#### Fixes issues
... 

#### Check the appropriate checkboxes below, please

- [ ] [BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/) 
- [ ] New feature 
- [x] I have read and signed [License Agreement for contributions](https://www.shopsys.com/license-agreement)

----
:globe_with_meridians: Live Preview:
  - https://tc-ssp-2808-tablet-account-popups.odin.shopsys.cloud
  - https://cz.tc-ssp-2808-tablet-account-popups.odin.shopsys.cloud

### CODE REVIEW

File: project-base/storefront/components/Layout/Header/Cart/CartInHeader.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const CartInHeader: FC = ({ className }) => {
 
     const isPriceVisibleOrEmtpyCart = isPriceVisible(cart?.totalItemsPrice.priceWithVat) || !cart?.items.length;
     const { width } = useGetWindowSize();
-    const isDesktop = width > desktopFirstSizes.tablet;
+    const isDesktop = width > mobileFirstSizes.vl;
```
Comment (JanMolcik) [inline project-base/storefront/components/Layout/Header/Cart/CartInHeader.tsx]: Let's use `useMediaMin` everywhere (if applicable) to unify it.

File: project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicItemUserAuthenticated.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const MenuIconicItemUserAuthenticated: FC = () => {
     const isHoveredDelayed = useDebounce(isHovered, 200);
 
     const { width } = useGetWindowSize();
-    const isDesktop = width > desktopFirstSizes.tablet;
+    const isDesktop = width > mobileFirstSizes.vl;
```
Comment (JanMolcik) [inline project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicItemUserAuthenticated.tsx]: Let's use `useMediaMin` everywhere (if applicable) to unify it.

**Review by JanMolcik — CHANGES_REQUESTED** [review]

File: project-base/storefront/components/Layout/Header/Cart/CartInHeader.tsx (hunk 1 above)
Comment (chlebektomas) [reply project-base/storefront/components/Layout/Header/Cart/CartInHeader.tsx]: fixed

File: project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicItemUserAuthenticated.tsx (hunk 2 above)
Comment (chlebektomas) [reply project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicItemUserAuthenticated.tsx]: fixed

**Review by JanMolcik — APPROVED** [review]

**Review by JanMolcik — APPROVED** [review]

**Review by JanMolcik — APPROVED** [review]

File: project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicElements.tsx [hunk 3]
```diff
@@ -96,15 +105,24 @@ export const MenuIconicItemUserPopover: FC<MenuIconicItemUserPopoverProps> = ({
...
+        : 'right-0 max-w-[335px] lg:right-[-180px] lg:min-w-[740px] vl:min-w-[807px]';
 
     return (
         <AnimatePresence initial={false}>
             {isHovered && (
                 <AnimateAppearDiv
                     className={twMergeCustom(
-                        `pointer-events-auto absolute ${positionClasses} top-[54px] z-cart hidden ${isAuthenticated ? 'min-w-[355px]' : 'max-w-[335px] lg:right-[-180px] lg:min-w-[740px] vl:min-w-[807px]'} origin-top`,
-                        'rounded-xl bg-background p-5 lg:block',
+                        `pointer-events-auto absolute ${positionClasses} top-[54px] z-cart hidden origin-top`,
```
Comment (JanMolcik) [inline project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicElements.tsx]: `positionClasses` should be added as second parameter to `twMergeCustom` in my opinion.

**Review by JanMolcik — CHANGES_REQUESTED** [review]

File: project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicElements.tsx (hunk 3 above)
Comment (chlebektomas) [reply project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicElements.tsx]: fixed

**Review by JanMolcik — APPROVED** [review]

**Pull request merged**